localstripe/resources.py

```diff
@@ -475,6 +475,33 @@ def amount_refunded(self):
     def refunded(self):
         return self.amount <= self.amount_refunded
 
+    @classmethod
+    def _api_list_all(cls, url, customer=None, created=None, limit=10):
+        try:
+            assert _type(customer) is str and customer.startswith('cus_')
+            if type(created) is dict:
+                assert len(created.keys()) == 1 and \
+                    list(created.keys())[0] in ('gt', 'gte', 'lt', 'lte')
+                date = try_convert_to_int(list(created.values())[0])
+                assert type(date) is int and date > 1500000000
+            elif type(created) is str:
+                date = try_convert_to_int(created)
+                assert type(date) is int and date > 1500000000
+            elif created:
+                raise UserError(400, 'Bad request')
```

Comment:
To be consistent with the rest of the code, I'd prefer
```python
assert type(created) in (dict, str)
```
on line 482.

Reply:
Nice idea!
Here is the git diff I push to fix it:
```diff
--- a/localstripe/resources.py
+++ b/localstripe/resources.py
@@ -479,16 +479,14 @@ class Charge(StripeObject):
     def _api_list_all(cls, url, customer=None, created=None, limit=10):
         try:
             assert _type(customer) is str and customer.startswith('cus_')
+            assert type(created) in (dict, str)
             if type(created) is dict:
                 assert len(created.keys()) == 1 and \
                     list(created.keys())[0] in ('gt', 'gte', 'lt', 'lte')
                 date = try_convert_to_int(list(created.values())[0])
-                assert type(date) is int and date > 1500000000
             elif type(created) is str:
                 date = try_convert_to_int(created)
-                assert type(date) is int and date > 1500000000
-            elif created:
-                raise UserError(400, 'Bad request')
+            assert type(date) is int and date > 1500000000
         except AssertionError:
             raise UserError(400, 'Bad request')